### PR TITLE
Update install command for etcdctl

### PR DESCRIPTION
**Pull request by dereknola**
Signed-off-by: Derek Nola <derek.nola@suse.com>

The k3s info on how to install etcdctl (written by me) does not correctly move etcdctl into a user's PATH. This is now fixed.

### CODE REVIEW

**Review by brandond — APPROVED** [review]

File: content/k3s/latest/en/advanced/_index.md [hunk 1]
```diff
@@ -126,9 +126,8 @@ etcdctl provides a CLI for etcd.
 If you would like to use etcdctl after installing K3s with embedded etcd, install etcdctl using the [official documentation.](https://etcd.io/docs/latest/install/) 
 
 ```
-$ VERSION="v3.5.0"
-$ curl -L https://github.com/etcd-io/etcd/releases/download/${VERSION}/etcd-${VERSION}-linux-amd64.tar.gz --output etcdctl-${VERSION}-linux-amd64.tar.gz
-$ sudo tar -zxvf etcdctl-${VERSION}-linux-amd64.tar.gz -C /usr/local/bin
+$ VERSION="v3.5.0" curl -L https://github.com/etcd-io/etcd/releases/download/${VERSION}/etcd-${VERSION}-linux-amd64.tar.gz --output etcdctl-linux-amd64.tar.gz
+$ sudo tar -zxvf etcdctl-linux-amd64.tar.gz --strip-components=1 -C /usr/local/bin etcd-v3.5.0-linux-amd64/etcdctl
```
Comment (btat) [inline content/k3s/latest/en/advanced/_index.md:130]: @dereknola Does the v3.5.0 need to be hardcoded here or can ${VERSION} be used as well?

Comment (dereknola) [reply content/k3s/latest/en/advanced/_index.md:130]: Well the `v.3.5.0` is the VERSION being defined... so you can't use ${VERSION}, as then there would be no version defined at all.

Comment (btat) [reply content/k3s/latest/en/advanced/_index.md:130]: I meant on the `etcd-v3.5.0-linux-amd64/etcdctl` part.

Comment (dereknola) [reply content/k3s/latest/en/advanced/_index.md]: Resolved

**Pull request merged**